Patches/InputTweaks.cpp

```diff
@@ -805,15 +934,15 @@ bool InputTweaks::SetRMBAimFunction()
 		!IsInFullScreenImageEvent()));
 }
 
-bool InputTweaks::IsInFullScreenImageEvent()
+bool IsInFullScreenImageEvent()
```

Comment:
Why not move this function into Common.cpp?

Reply:
Sure, give me some time and I'll move it there!

Reply:
@elishacloud moved the function to common.cpp!